CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+## Developer Git Workflow:
+
+- 📌 Clone down the organization's repo as your `origin`
+- 📌 Pick up a "ticket" from our [GitHub KanBan Board](https://github.com/orgs/QueerGlobal/projects/1/views/2) `Todo` lane
+  - 🐱‍🏍 Move the "ticket" into the `In Progress` lane
+  - 🏂 Click the title of the "ticket" and in the top right section make yourself an "Assignee"
+  - 💃 Click the title again for that ticket's _full_ view on GitHub
+  - 🏃‍♀️ Click "create a branch" directly on the right side under "development"
```

Comment:
Is this a soft intro to GitHub Flow?

Reply:
Not sure I totally know what you mean, but I think my answer is yes?